server/app/routes.py

```diff
@@ -112,3 +116,102 @@ def delete_user(user_id):
         "message": "User deleted successfully.",
         "data": None
     }), 200
+
+# Invitation routes
+
+
+@app.route('/invitations', methods=['GET'])
+def get_invitations():
+    invitations = Invitation.query.all()
+    return jsonify([invitation.to_dict() for invitation in invitations])
+
+
+@app.route('/invitations/<int:invitation_id>', methods=['GET'])
+def get_invitation(invitation_id):
+    invitation = Invitation.query.get_or_404(invitation_id)
+    return jsonify(invitation.to_dict())
```

Comment:
Keep the response format consistent so that it maintains a consistent structure.
Use this as the reference:
```python
return jsonify({
            "status": "Failed",
            "message": "Email and user_id are required.",
            "data": None
        }), 400
```

Reply:
Formatted and updated GET list_invitation()